tracer/src/Datadog.Tracer.Native/debugger_constants.h

```diff
@@ -24,7 +24,8 @@ const WSTRING skip_assembly_prefixes[]{
     WStr("csc"),
     WStr("DuckTypeNotVisibleAssembly"),
     WStr("Datadog.Trace"),
-    WStr("Newtonsoft.Json")};
+    WStr("Newtonsoft.Json"),
+    WStr("xunit")};
```

Comment:
This may disable CI Visibility, I guess this is fine if only affects the debugger, but we should be careful.